src/SFA.DAS.EAS.Portal.Client/Models/Concrete/AccountLegalEntityDto.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace SFA.DAS.EAS.Portal.Client.Models.Concrete
+{
+    public class AccountLegalEntityDto : IAccountLegalEntityDto
+    {
+        public AccountLegalEntityDto(IEnumerable<ReservedFundingDto> reservedFundings)
+        {
+            ReservedFundings = reservedFundings;
+        }
+
+        [JsonProperty("accountLegalEntityId")]
+        public virtual long AccountLegalEntityId { get; private set; }
+
+        [JsonProperty("legalEntityName")]
+        public virtual string LegalEntityName { get; private set; }
+        
+        [JsonProperty("reservedFundings")]
+        public virtual IEnumerable<IReservedFundingDto> ReservedFundings { get; private set; }
```

Comment:
These are Reservations.  That is the domain the events that create them come from. Already in Ashley's Account read model.